### ISSUE TITLE
A warning appears during the running of the program, will this affect the accuracy?

### ISSUE THREAD

**Issue by useunreal** [voldemortX/DST-CBC#4]
Warning:  multi_tensor_applier fused unscale kernel is unavailable, possibly because apex was installed without --cuda_ext --cpp_ext. Using Python fallback.  Original ImportError was: ModuleNotFoundError("No module named 'amp_C'",)

The version of pytorch I installed is 1.2.0 and the version of torchvision is 0.4.0，and  the version of apex is 0.1

**Comment by voldemortX** [comment]
Your apex installation could be incorrect.  Although it is only a warning, I am not sure if this won't affect mix precision training.

**Comment by useunreal** [comment]
Thank you, I will try again to reinstall apex
